### DISCUSSION TITLE
Pick an SPA Framework

### DISCUSSION THREAD

**Discussion by markdav-is** [OpenEugene/fire-refuge#4] [Ideas]
If I were going to do this by myself, I'd use Blazor.  But that's not as popular as React or Angular or Vue.
React might be the best choice for attracting the most developers to the proejct.

**Comment by mckelveygreg** [comment]
I'd be able to get going quickly with React. 
Other decisions to make would be:
- Typescript?
- UI lib (ex: Material UI)
- React framework (ex: Gatsby, Next, or Create React App)

**Reply by markdav-is** [reply]
I'd say go for whatever gives you the most joy and will attract contributors.

**Reply by joshmarinacci** [reply]
I would go with create-react-app. It's extremely common and it's bugs well known. Almost anyone will be able to use it.

**Comment by markdav-is** [comment]
we ended up using Blazor because we had lots of core contributors from Dark Matter.
